license: nodes number check (#1167)
vesoft-inc/nebula-docs: docs-2.0/4.deployment-and-installation/deploy-license.md

{

>   "vendor": "Vesoft_Inc",

Reviewer: please check the license info. Should it be "vesoft"?

|`product`|The product type. The product type of Nebula Graph is `nebula_graph`.|
|`version`|The version information.|
|`licenseType`|The license type, including `enterprise`, `samll_bussiness`, `pro`, and `individual`. |

> |`graphdSpec`| The limitation of the number of graph services in a cluster. Nebula Graph detects the number of graph services currently active in real-time, the graph services that exceed the limit can not connect to the cluster. |

Reviewer: ```suggestion
|`graphdSpec`| The limitation of the number of graph services in a cluster. Nebula Graph detects the number of active graph services in real-time. You are unable to connect to the cluster once the number exceeds the limit. |
```

> |`storagedSpec`| The limitation of the number of storage services in a cluster. Nebula Graph detects the number of storage services currently active in real-time, the storage services that exceed the limit can not connect to the cluster. |

Reviewer: ```suggestion
|`storagedSpec`| The limitation of the number of storage services in a cluster. Nebula Graph detects the number of active storage services in real-time. You are unable to connect to the cluster once the number exceeds the limit. |
```
